src/main/webapp/index.html

```diff
@@ -22,7 +22,14 @@ <h1>
           Discover the world's hidden gems <br />
           before planning your next trip.
         </h1>
-        <input type="text" name="search" placeholder="Search a city, country, etc." />
+        
+        <!-- Search Bar -->
+        <form autocomplete="off">
```

Comment:
To get this to redirect to `SearchResults.html` I think this should be:
```suggestion
        <form autocomplete="off" action="/SearchResults.html" method="GET">
```